ext/drmgr/drmgr.h

```diff
@@ -785,6 +806,17 @@ bool
 drmgr_register_thread_exit_event_ex(void (*func)(void *drcontext),
                                     drmgr_priority_t *priority);
 
+DR_EXPORT
+/**
+ * Registers a callback function for the thread exit event,
+ * ordered by \p priority. Allows for the passing of user data \p usr_data
+ * which is available upon the execution of the callback. drmgr calls \p func
+ * when an application thread exits. \return whether successful.
+ */
+bool
+drmgr_register_thread_exit_event_usr_data(void (*func)(void *drcontext, void *usr_data),
```

Comment:
Ditto on the test and release.dox feature line (I'd combine with the init one in one line)